test-harness/specs/security-scheme-validation-AC-12.only_oas3.txt

```diff
@@ -0,0 +1,58 @@
+====test====
+After a user obtains a cookie from /login, s/he can then use the cookie to access protected endpoint /todos.
```

Comment:
Assumed: since it's relatively easy to create an endpoint that returns *Set-Cookie* header,
it would be OK to actually compare *Set-Cookie* content with what is sent later. So, basically, if
SESSIONID from /login response (*Set-Cookie*) and SESSIONID from a request to /todos (*Cookie*) match.